infra.bs

```diff
@@ -542,6 +543,74 @@ actually ends up representing <a lt="JavaScript string">JavaScript</a> and
 <a>scalar value strings</a>. It is even fairly typical for implementations to have multiple
 implementations of just <a>JavaScript strings</a> for performance and memory reasons.
 
+<hr>
+
+<p>A <a>string</a> <var>a</var> is a
+<dfn export lt="code unit prefix|starts with">code unit prefix</dfn> of a <a>string</a> <var>b</var>
+if the following steps return true:
+
+<ol>
+ <li><p>Let <var>i</var> be 0.
+
+ <li>
+  <p><a>While</a> true:
+
+  <ol>
+   <li><p>Let <var>aCodeUnit</var> be the <var>i</var>th <a>code unit</a> of <var>a</var>, or null
+   if <var>i</var> is greater than or equal to <var>a</var>'s <a for=string>length</a>.
```

Comment:
How about applying https://github.com/whatwg/infra/issues/247#issuecomment-488227069 here and below as a start?